source/features/hide-trends-and-who-to-follow.js

```diff
@@ -1,6 +1,8 @@
 import elementReady from 'element-ready';
 
 export default async function () {
-	const sidebar = await elementReady('[data-testid=sidebarColumn]');
-	$(sidebar).hide();
+	const trending = await elementReady('[aria-label="Timeline: Trending now"]');
+	$(trending).hide();
+	const whoToFollow = await elementReady('[class="css-1dbjc4n r-1u4rsef r-9cbz99 r-1867qdf r-1phboty r-rs99b7 r-ku1wi2 r-1bro5k0 r-1udh08x"]');
```

Comment:
I would try to find the minimal amount of classes that are unique to this element, as this selector is super fragile since it's so specific.

Reply:
I see. I will try to find another way to limit the selector. Thank you.